src/layouts/Banner/BannerLayout.tsx

```diff
@@ -39,7 +39,9 @@ export const BannerLayout = ({ marketplace, children }: BannerLayoutProps) => {
           <div className="flex items-center justify-end">
             {equals(
               publicKey?.toBase58(),
-              marketplace.auctionHouse.authority
+              marketplace.auctionHouses
+                ? marketplace.auctionHouses[0].authority
+                : null
```

Comment:
Since we are running the backfill to move all auctionHouse to auctionHouses this should always exist. This should be able to just be `marketplace.auctionHouses[0].authority`